handlers/dashboard.go

```diff
@@ -988,7 +988,7 @@ func DashboardDataEffectiveness(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if len(activeValidators) == 0 {
-		http.Error(w, "Invalid query", http.StatusBadRequest)
+		w.WriteHeader(http.StatusNoContent)
```

Comment:
Incorrect usage of `StatusNoContent`. Also I'm not sure `w.WriteHeader` actually produces a response, just that it changes what the response code will be once it does happen, pls confirm.

Reply:
Thanks for your valuable input, I have adapted the code based on your suggestions.

Reply:
Thanks for your valuable response, I have resolved this conversation based on your adaptations.